File: BLL/DTO/ForumTopicDto.cs
using System;
using System.Collections.Generic;

namespace BLL.DTO
{
    public class ForumTopicDto
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public DateTime Date { get; set; }
        public Guid UserId { get; set; }
        public UserDto User { get; set; }
        public Guid ForumCategoryId { get; set; }
        public ForumCategoryDto ForumCategory { get; set; }
        public List<CommentDto> Comments { get; set; }
    }
}

File: JokrStore-SPA/src/app/_models/forumTopicDto.ts
/* Auto Generated */

import { UserDtoLite } from "./UserDtos/userDtoLite";
import { ForumCategoryDto } from "./forumCategoryDto";

export interface ForumTopicDto {
    id: any;
    title: string;
    content: string;
    date: Date;
    userId: any;
    user: UserDtoLite;
    forumCategoryId: any;
    forumCategory: ForumCategoryDto;
    comments: any[];
}
